python/doc/theory/probabilistic_modeling/process_definitions.rst

```diff
@@ -27,18 +30,25 @@ In this document, we note:
    :math:`m(\vect{t})=\Expect{X_{\vect{t}}}`,
 
 -  :math:`C : \cD \times \cD \rightarrow  \cM_{d \times d}(\Rset)` its
-   *covariance function*, defined by
-   :math:`C(\vect{s}, \vect{t})=\Expect{(X_{\vect{s}}-m(\vect{s}))(X_{\vect{t}}-m(\vect{t}))^t}`,
+   *covariance function*, defined by:
+
+.. math::
+   \operatorname{Cov}(\vect{s}, \vect{t})
```

Comment:
`\Cov` is defined in math_norations.sty and should be used here.

Reply:
Arf, yes and no. Fixed at f6345cc0d.

Here is the new version:
![image](https://github.com/openturns/openturns/assets/31351465/dea06634-8314-413c-9b0e-e2c33692e19e)